Makefile

```diff
@@ -41,8 +44,9 @@ HADOLINT_VERSION ?= v2.8.0
 HADOLINT_BIN := out/linters/hadolint-$(HADOLINT_VERSION)-$(LINT_ARCH)
 $(HADOLINT_BIN):
 	mkdir -p out/linters
-	rm -rf out/linters/hadolint-*
-	curl -sfL https://github.com/hadolint/hadolint/releases/download/v2.6.1/hadolint-$(LINT_OS)-$(LINT_ARCH) > $@
+	curl -o $@.dl -sfL https://github.com/hadolint/hadolint/releases/download/$(HADOLINT_VERSION)/hadolint-$(LINT_OS)-$(LINT_ARCH) \
```

Comment:
I just noticed this is missing the `-S` that the shellcheck curl has, would be good to keep the same flags (ideally in the same order too (I'd say `-sSfL -o` that way the common stuff is first) )

Reply:
Fixed!